src/views/pages/settings/Settings.language.page.tsx

```diff
@@ -0,0 +1,86 @@
+import React, { useState } from 'react';
+import { useTranslation } from 'react-i18next';
+import { PageEditor, PageTemplate } from '@components/pages';
+import { Label, RadioInput } from '@components/inputs';
+import i18n from '@src/i18n';
+import styled from 'styled-components';
+
+const STUDIO_LANGUAGES = ['de', 'es', 'en', 'fr', 'it', 'pt'] as const;
+const STUDIO_LANGUAGE_NAMES = {
+  en: 'English, US',
+  fr: 'Français, FR',
+  es: 'Español',
+  it: 'Italiano',
+  de: 'Deutsch',
+  pt: 'Português',
```

Comment:
Do we already support all those languages?

Reply:
Oui mais partiellement. Quand l'environnement pour la traduction de Studio sera fini, on ajoutera un texte dans cette UI pour indiquer aux makers où ils peuvent contribuer afin de terminer les langues incomplètes ou ajouter de nouvelles langues.